src/Model/Profile.php

```diff
@@ -341,7 +341,7 @@ public static function getVCardHtml(array $profile, bool $block, bool $show_cont
 				if ($visitor_is_following) {
 					$unfollow_link = $visitor_base_path . '/contact/unfollow?url=' . urlencode($profile_url) . '&auto=1';
 				} else {
-					$follow_link =  $visitor_base_path . '/follow?url=' . urlencode($profile_url) . '&auto=1';
+					$follow_link =  $visitor_base_path .'/contact/follow?url=' . urlencode($profile_url) . '&auto=1';
```

Comment:
Not sure how this got past PHP CodeSniffer
```suggestion
					$follow_link =  $visitor_base_path . '/contact/follow?url=' . urlencode($profile_url) . '&auto=1';
```